src/meta/meta_bulk_load_service.cpp

```diff
@@ -450,6 +455,18 @@ void bulk_load_service::partition_bulk_load(const std::string &app_name, const g
 
     bulk_load_rpc rpc(std::move(req), RPC_BULK_LOAD, 0_ms, 0, pid.thread_hash());
     rpc.call(primary_addr, _meta_svc->tracker(), [this, rpc](error_code err) mutable {
+
+        // fill host_port struct if needed
```

Comment:
The comment seems not enough, you need to mention the remote server doesn't support FQDN?